modules/core/src/directx.cpp

```diff
@@ -1494,6 +1495,7 @@ void getDeviceIDsByD3D11Device(ID3D11Device* pD3D11Device, std::vector<std::pair
                                   std::make_move_iterator(temp_devices.end()));
         }
     }
+    return std::move(dst);
```

Comment:
It brakes RVO, please return vector without move

Reply:
Removed